pkg/git/git_test.go

```diff
@@ -147,20 +158,107 @@ func Test_validateGitAuth(t *testing.T) {
 			credsDir, cleanup := createTempDir(t)
 			defer cleanup()
 			if tt.wantSSHdir {
-				os.MkdirAll(credsDir+"/.ssh", 0755)
+				err := os.MkdirAll(filepath.Join(credsDir, ".ssh"), fileMode)
+				if err != nil {
+					t.Errorf("Error creating SSH dir: %v", err)
```

Comment:
should we "fatal" here as.. if the creation failed, we may not need to continue that subtest ?